src/ng/compile.js

```diff
@@ -1319,13 +1318,6 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
         // modify it.
         $compileNodes = jqLite($compileNodes);
       }
-      // We can not compile top level text elements since text nodes can be merged and we will
-      // not be able to attach scope data to them, so we will wrap them in <span>
```

Comment:
There are two parts:
* In theory, you can break things up if you do a *hand* transclude and then call `normalize` on an ascendant node. All the built-in directives do not suffer from this issue as they add a comment node between in between the transcluded content
* Given a text node that is a root node of transcluded content, you will not be able to call `scope` to get the element scope. Old parts of the code relied on this, but this is no longer true